includes/Modules/Search_Console.php

```diff
@@ -247,33 +245,28 @@ protected function parse_data_response( Data_Request $data, $response ) {
 		switch ( "{$data->method}:{$data->datapoint}" ) {
 			case 'GET:matched-sites':
 				/* @var Google_Service_Webmasters_SitesListResponse $response Response object. */
-				$sites            = $this->map_sites( (array) $response->getSiteEntry() );
-				$current_url      = $this->context->get_reference_site_url();
-				$current_host     = wp_parse_url( $current_url, PHP_URL_HOST );
-				$property_matches = array_filter(
-					$sites,
-					function ( array $site ) use ( $current_host ) {
-						$site_host = wp_parse_url( str_replace( 'sc-domain:', 'https://', $site['siteURL'] ), PHP_URL_HOST );
-
-						// Ensure host names overlap, from right to left.
-						return 0 === strpos( strrev( $current_host ), strrev( $site_host ) );
-					}
-				);
+				$sites = $this->map_sites( (array) $response->getSiteEntry() );
 
-				$exact_match = array_reduce(
-					$property_matches,
-					function ( $match, array $site ) use ( $current_url ) {
-						if ( ! $match && trailingslashit( $current_url ) === trailingslashit( $site['siteURL'] ) ) {
-							return $site;
-						}
-						return $match;
-					},
-					null
+				$current_url                  = trailingslashit( $this->context->get_reference_site_url() );
+				$sufficient_permission_levels = array(
+					'siteRestrictedUser' => true,
+					'siteOwner'          => true,
+					'siteFullUser'       => true,
```

Comment:
Why not use a simple list and check if `in_array()` below?

Reply:
`isset()` is faster than `in_array()`, so I tend to use that where it's easy enough.

Reply:
In this case I think it's more of a preference thing since the difference in performance is probably zero with such a small set. In cases like these I prefer to opt on the side of readability and clarity of intent, not just for the condition itself but the code overall.